committing.rst

```diff
@@ -235,10 +251,10 @@ it is common to simply paste the NEWS entry into the commit message.  Here
 is an example::
 
    bpo-42: the spam module is now more spammy.
-   
+
    The spam module sporadically came up short on spam. This change
    raises the amount of spam in the module by making it more spammy.
-   
+
```

Comment:
If you mean the indent, that's once again because it's reST formatting for a literal block. If you mean the newline then I personally have always used it.

Regardless, I didn't change this format and so if you want to discuss the formatting of commit messages that belongs in a separate issue.